freshmaker/pyxis.py

```diff
@@ -169,19 +169,24 @@ def get_latest_bundles(self, index_images):
             request_params['filter'] = \
                 f'latest_in_channel==true and source_index_container_path=={path}'
 
-            def _isvalid(version):
+            def _isvalid(version, csv_name):
```

Comment:
so the `csv_name` parameter needed only for printing an error message in case of exception. Seems kind of weird.

Reply:
@apaplaus it's a small helper function, so I wouldn't be too considered if it seems a little weird. That being said, I could pass in the whole bundle instead.

Reply:
Ok, if it isn't a problem for you we can leave it as it is.